pkg/api/validation/validation_test.go

```diff
@@ -6300,7 +6300,7 @@ func TestValidateConfigMap(t *testing.T) {
 		invalidName      = newConfigMap("NoUppercaseOrSpecialCharsLike=Equals", "validns", nil)
 		emptyNs          = newConfigMap("validname", "", nil)
 		invalidNs        = newConfigMap("validname", "NoUppercaseOrSpecialCharsLike=Equals", nil)
-		invalidKey       = newConfigMap("validname", "validns", map[string]string{"a..b": "value"})
```

Comment:
Why?  What value does it provide or what does it protect against?

On Mon, Aug 1, 2016 at 2:39 PM, Paul Morie <notifications@github.com> wrote:

> In pkg/api/validation/validation_test.go
> <https://github.com/kubernetes/kubernetes/pull/25458#discussion_r73057977>
> :
>
> > @@ -6300,7 +6300,7 @@ func TestValidateConfigMap(t *testing.T) {
> >  		invalidName      = newConfigMap("NoUppercaseOrSpecialCharsLike=Equals", "validns", nil)
> >  		emptyNs          = newConfigMap("validname", "", nil)
> >  		invalidNs        = newConfigMap("validname", "NoUppercaseOrSpecialCharsLike=Equals", nil)
> > -		invalidKey       = newConfigMap("validname", "validns", map[string]string{"a..b": "value"})
>
> i think that two dots next to one another should still be invalid.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25458/files/d4969ff03230e61fcd18fef778debd83e896ed6a#r73057977>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVFYXhORgQ3rm51RN1dSuuwcTYTlCks5qbmd8gaJpZM4IbwmI>
> .
>